src/cadenza.js

```diff
@@ -291,7 +291,7 @@ export class CadenzaClient {
    * @param {AbortSignal} [options.signal] - A signal to abort the iframe loading
    * @return {Promise<void>} A `Promise` for when the iframe is loaded
    * @throws For invalid arguments
-   * @fires {@link CadenzaDrillThroughEvent}
+   * @fires {@link CadenzaDrillThroughEvent | CadenzaActionEvent}
```

Comment:
Does this show up as individual links to the docs of these classes in the docs? Otherwise please do it like e.g. with the #editGeometry events.